src/lxc/tools/lxc_autostart.c

```diff
@@ -311,6 +311,8 @@ static int get_config_integer(struct lxc_container *c, char *key) {
 	if (lxc_safe_int(value, &ret) < 0)
 		DEBUG("Could not parse config item.");
 
+	free(value);
```

Comment:
Hm, I see.  No, I'd really really prefer rebasing to fix the original patches.  Keeping the tree more bisectable.